src/store/courses/reducer.js

```diff
@@ -0,0 +1,22 @@
+import { SET_COURSES, CREATE_COURSE, DELETE_COURSE } from './actionTypes';
+
+const coursesInitialState = [];
+
+export const coursesReducer = (state = coursesInitialState, action) => {
+	switch (action.type) {
+		case SET_COURSES:
```

Comment:
no possibility to update course and get all courses

Reply:
The 'Update course' functionality will be added in the next module.
The 'Get all courses' functionality  works in useEffect hook in the Courses.jsx. It will be refactored and added to the reducer in the next module with Redux thunk.